src/transaction/txParser.ts

```diff
@@ -248,11 +283,35 @@ const parseTxCerts = (txCertificates: any[]): _Certificate[] => {
   )
 }
 
+const parseTxWithdrawalRewardAccount = (addressBuffer: Buffer): StakeCredentials => {
+  const address = Cardano.Address.from_bytes(addressBuffer)
+  const type = getAddressType(address)
+  switch (type) {
+    case (AddressType.REWARD_KEY): {
+      return {
+        type: StakeCredentialsKeys.ADDR_KEY_HASH,
+        addrKeyHash: rewardAddressToPubKeyHash(addressBuffer),
+      }
+    }
+    case (AddressType.REWARD_SCRIPT): {
+      return {
+        type: StakeCredentialsKeys.SCRIPT_HASH,
+        scriptHash: addressBuffer,
```

Comment:
The `rewardAddressToPubKeyHash` should be used here also. Perhaps we can rename `rewardAddressToPubKeyHash` to `rewardAddressToStakeCredential` and reuse it here.